tools/tests/scrollable-container/dynamic-content.test.js

```diff
@@ -0,0 +1,119 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+function showDynamicScrollbarContent(scrollableContainer) {
+    scrollableContainer.querySelector('.dynamic-content').style.display = 'block';
+}
+
+function hideDynamicScrollbarContent(scrollableContainer) {
+    scrollableContainer.querySelector('.dynamic-content').style.display = 'none';
+}
+
+describe('Scrollable Container Component', () => {
+    afterAll(() => cleanTestPage('.scrollable-container-test-wrapper'));
+
+    beforeEach(function (done) {
+        const template = `<gameface-scrollable-container class="scrollable-container" automatic>

```

Comment:
Is there any reason to not define the template globally?

Reply:
Scope. If I declare it as a global I have to make sure its name is unique. Surely another test has a template named template.

Reply:
Do we want to re-create a new variable with the template before each test starts? Can we move this template up inside the describe scope and will it work?

Reply:
Moved.